packages/common/src/ui/layout/tables/components/index.tsx

```diff
@@ -5,11 +5,14 @@ import { useTranslation } from '../../../../intl';
 
 export * from './DataRow';
 
+const capitalize = (str: string) =>
+  str.slice(0, 1).toUpperCase() + str.slice(1, str.length).toLowerCase();
+
 export const BasicCell = <T extends DomainObject>({
   column,
   rowData,
 }: CellProps<T>): ReactElement => {
-  return <>{column.accessor(rowData)}</>;
+  return <>{capitalize(String(column.accessor(rowData)))}</>;
```

Comment:
I added this as a cheap hack to make the statuses show nicely in the table just in case the video is made.. but I've made #275 to handle this better

Reply:
perhaps a `text-transform: capitalize;` would be enough?